apps/builder/app/builder/features/settings-panel/props-section/animation/animation-keyframes.tsx

```diff
@@ -125,21 +131,28 @@ const Keyframe = ({
   onChange: (value: AnimationKeyframe | undefined) => void;
 }) => {
   const ids = useIds(["offset"]);
+  const apiRef = useRef<CssEditorApi>();
 
-  const cssProperties = useMemo(() => {
-    let result = ``;
-    for (const [property, style] of Object.entries(value.styles)) {
-      result = `${result}${toKebabCase(property)}: ${toValue(style)};\n`;
-    }
-    return result;
-  }, [value.styles]);
+  const styleMap: CssStyleMap = useMemo(
+    () =>
+      new Map(
+        Object.keys(value.styles).map((key) => [
+          key as CssProperty,
```

Comment:
untyped by design?

Reply:
I don't like the entire value type def, but its made by @istarkov and I am not sure if its some temp thing, needs to be discussed, I am not 100% in favor of Map/Set and avoid using objects.

Reply:
e.g. we already use CssStyleMap in some cases, maybe that can be used instead here as well?

Reply:
CSSPropery should gone from type system. Not a lot of value, and it's very very long union type what makes ts slow.

Reply:
If we want to make it typed, we should use opaque types for unions like this.